internal/database/database.go

```
@@ -16,13 +21,195 @@ func Init(path string) (*sql.DB, error) {
+			firstErr = err
+		}
+	}
+	return firstErr
+}
+
+func copyFile(src, dst string) error {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `in.Close` is not checked

(errcheck)